.github/scripts/update_versions/update.py

```diff
@@ -0,0 +1,56 @@
+import json
+import requests
+# Gets data from apiUrl argument and check if current version is outdated and returns new versions if applicable
+def checkVersion(name, apiUrl, currentVersion):
+    # make Http Get request
+    r = requests.get(apiUrl)
+
+    #checks if request is successful
+    if r.status_code == 200:
+
+        # get data from request
+        request_version = r.json()[0]['latest']
```

Comment:
As discussed [here](https://github.com/Checkmarx/kics/pull/5358#discussion_r878156912), we should keep the latest stable version. Does the API URL chosen points only stable releases?

Reply:
In the end-of-life API not in pre-release or in-development releases appear, so I think the returned value is what we expect.